src/shared/sam/cli/samCliValidationNotification.ts

```diff
@@ -97,46 +100,56 @@ export function makeSamCliValidationNotification(
         )
     } else if (samCliValidationError instanceof InvalidSamCliVersionError) {
         return onCreateNotification(
-            makeVersionValidationNotificationMessage(samCliValidationError.versionValidation),
-            makeVersionValidationActions(samCliValidationError.versionValidation.validation)
+            getInvalidVersionMsg(samCliValidationError.versionValidation),
+            getActions(samCliValidationError.versionValidation.validation)
         )
     } else {
         return onCreateNotification(
             localize(
                 'AWS.samcli.notification.unexpected.validation.issue',
-                'An unexpected issue occured while validating SAM CLI: {0}',
+                'Unexpected error while validating SAM CLI: {0}',
                 samCliValidationError.message
             ),
             []
         )
     }
 }
 
-function makeVersionValidationNotificationMessage(validationResult: SamCliVersionValidatorResult): string {
-    const recommendationUpdateToolkit: string = localize(
-        'AWS.samcli.recommend.update.toolkit',
-        'Check the Marketplace for an updated {0} Toolkit.',
-        getIdeProperties().company
+function getInvalidVersionMsg(validationResult: SamCliVersionValidatorResult): string {
+    const win185msg = localize(
+        'AWS.sam.updateSamWindows',
+        'SAM CLI 1.85-1.86 has [known issues](https://github.com/aws/aws-sam-cli/issues/5243) on Windows. Update SAM CLI.'
     )
-
-    const recommendationUpdateSamCli: string = localize('AWS.samcli.recommend.update.samcli', 'Update your SAM CLI.')
-
-    const recommendation: string =
-        validationResult.validation === SamCliVersionValidation.VersionTooHigh
-            ? recommendationUpdateToolkit
-            : recommendationUpdateSamCli
+    let recommendation: string
+
+    switch (validationResult.validation) {
+        case SamCliVersionValidation.VersionTooHigh:
+            recommendation = localize('AWS.sam.updateToolkit', 'Update {0} Toolkit.', getIdeProperties().company)
+            break
+        case SamCliVersionValidation.Version185Win:
+            return win185msg
+        case SamCliVersionValidation.VersionNotParseable: {
+            if (process.platform === 'win32') {
+                return win185msg
```

Comment:
If sam is found but `sam --info` fails, show `SAM CLI 1.85-1.86 has [known issues](https://github.com/aws/aws-sam-cli/issues/5243) on Windows. Update SAM CLI.`